pkg/cli/cmd/recipe/register/register_test.go

```diff
@@ -138,7 +138,7 @@ func Test_Run(t *testing.T) {
 			ds_ctrl.MongoDatabasesResourceType: {
 				"cosmosDB": &v20231001preview.BicepRecipeProperties{
 					TemplateKind: to.Ptr(recipes.TemplateKindBicep),
-					TemplatePath: to.Ptr("testpublicrecipe.azurecr.io/bicep/modules/mongodatabases:v1"),
+					TemplatePath: to.Ptr("ghcr.io/testpublicrecipe/bicep/modules/mongodatabases:v1"),
```

Comment:
You can define ghcr.io/testpublicrecipe at one common util test pkg or define once in the test file and use it to avoid inline updates.

Reply:
This is applicable to all the _test files.

Reply:
agreed. I plan to add this as a fast follow to this PR